docs/entropic-workflow.md

```diff
@@ -0,0 +1,36 @@
+# A workflow for Entropic
+
+This project has to this point been a close collaboration between two people who worked together for several years, and have an established workflow. If we want to succeed with this project, we'll need a successful workflow for collaborating asynchronously with people who don't know each other as well. I have some suggestions based on both things I've done with teams in the past and descriptions of other successful processes.
+
+In this one-pager I propose a workflow and discuss some of the tradeoffs.
+
+## Overview
+
+There is no process that will replace talking to people. Write a problem statement and proposed solution. Share it privately with one or more contributors and ask for feedback ahead of starting a public discussion. Be most formal when the problem is especially tricky or contentious.
```

Comment:
How do people know who the contributors should be sending things privately to, are?
How do folk know how to contact these people privately?

Reply:
This is an excellent question! My thinking right now is: we'll list project leads in the README with contact info. Right now that's Ceej or I. The instruction should be to contact one of the project leads via their listed contact info with the proposal & "who should I talk to?" This implies that project leads should provide a method of getting in contact. Project leads might direct the person with a question to another project member who has indicated that they wouldn't mind being looped in to handle questions, or give feedback directly.

This is kind of a digital version of approaching a co-worker with "hey, I was thinking about approaching problem $X by doing solution $Y. Who should I talk to? Does this make sense to you? What am I missing?" ahead of posting the issue to a team-wide forum. The goals are to get good, high-bandwidth feedback, end up with more people that can champion the work, and reduce the "why wasn't I consulted" effect.

Does this sound like a workable process? What gaps am I missing here?